backend/fms_core/utils.py

```diff
@@ -114,7 +114,24 @@ def convert_concentration_from_nm_to_ngbyul(concentration_nm, molecular_weight,
 
     return concentration
 
-def ensure_iterable(obj: Union[Any, None, Iterable[Any]]) -> Iterable[Any]:
+T = TypeVar("T")
```

Comment:
@UlysseFG Is this ok?

Reply:
eheh you can remove that TypeVar thing ... This does not help clarify things. Your comment give the gist of it ... I would personally call it "make_generator" or something of the kind but it's really a personal preference, You can merge when you are happy with what you have...